R/board_url.R

```diff
@@ -23,31 +23,50 @@
 #' @inheritParams new_board
 #' @export
 #' @examples
-#' github_raw <- "https://raw.githubusercontent.com/"
+#' github_raw <- function(x) paste0("https://raw.githubusercontent.com/", x)
+#'
 #' board <- board_url(c(
-#'   files = paste0(github_raw, "rstudio/pins-r/master/tests/testthat/pin-files/"),
-#'   rds = paste0(github_raw, "rstudio/pins-r/master/tests/testthat/pin-rds/"),
-#'   raw = paste0(github_raw, "rstudio/pins-r/master/tests/testthat/pin-files/first.txt")
+#'   files = github_raw("rstudio/pins-r/master/tests/testthat/pin-files/"),
+#'   rds = github_raw("rstudio/pins-r/master/tests/testthat/pin-rds/"),
+#'   raw = github_raw("rstudio/pins-r/master/tests/testthat/pin-files/first.txt")
```

Comment:
Add example using manifest?

Reply:
I don't think an example will be super useful because you need to serve the folder in some way to be able to use `board_url()`; in tests we use webfakes. Instead, let's do the vignette as described in #685 and be sure to point to the vignette in the docs for both `board_url()` and `write_board_manifest()`.

Reply:
So there's no way to use this functionality if you're informally publishing your files on Github?

Reply:
Yep, that is possible. I added some pins plus a manifest in #688 to serve from a GitHub directory.